manifests/s/sum1re/COCR/0.0.7-beta/sum1re.COCR.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: sum1re.COCR
+PackageVersion: 0.0.7-beta
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- cocr
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  Scope: user
+  InstallerUrl: https://github.com/sum1re/caption_ocr_tool/releases/download/0.0.7-beta/cocr_0.0.7-beta_windows_installer_x86_64.exe
+  InstallerSha256: F76BA982450DB4C721A8D0B2AE6AA4EAEEB42DFD4E2883EB8FC604818CF533E8
```

Let's add the `ProductCode` as there is no publisher listed in the ARP.

```suggestion
  InstallerSha256: F76BA982450DB4C721A8D0B2AE6AA4EAEEB42DFD4E2883EB8FC604818CF533E8
  ProductCode: "{533A06CF-B5E2-499B-8B3A-43CD54C1625B}_is1"
```